src/server/store/ReplyStore.ts

```diff
@@ -0,0 +1,48 @@
+import { AbstractLevel } from 'abstract-level'
+import { APActivity } from 'activitypub-types'
+
+export class ReplyStore {
```

Comment:
Instead of storing replies directly it should work more like the `forActor` API in the main Store. https://github.com/hyphacoop/social.distributed.press/blob/initial/src/server/store/index.ts#L35

So it should be more like this:

```JavaScript
export class ReplyStore {
repliesCache: Map<string, ActivityStore>

forPost(postURL) : ActivityStore {
  if (!this.repliesCache.has(postURL)) {
      const sub = this.db.sublevel(domain, { valueEncoding: 'json' })
      const store = new ActivityStore(sub)
      this.repliesCache.set(postURL, store)
    }

    const store = this.repliesCache.get(domain)
    if (store == null) {
      throw new Error('Domain store not initialixed')
    }
    return store
  }
}
add(reply) {
// get which message this is replying to
// get activitystore for it
// add reply there
}
list(postURL) {
// get store for replies for this post
// list all activities in it
}
}
```